omnichannel-server/src/feature/message-source/MessageParser.ts

```diff
@@ -3,10 +3,18 @@ import { MessageTwilioPayload } from "./interfaces/MessageTwilioPayload";
 import { MessagePayload } from "./interfaces/MessagePayload";
 import { MessageProvider } from "./interfaces/MessageProvider";
 
-export const MessageParser = ({ body }) => {
+export class MessageParser {
+
+messageSource(req: Request, res: Response):Promise<Response>{
+  const payload = this.messageParser(req)
+
+  return res.json({ status: "success" })
+}
+
+messageParser = ({ body }) => {
```

Comment:
Es mejor si aceptas de una vez el body, en vez del request completo: 
```suggestion
parseRequestBody = (body) => {
```